landing-page/types/headnode/layouts/navbar.html.erb

```diff
@@ -61,16 +61,39 @@
             </li>
 
             <% unless app_items.empty? %>
-              <li class="nav-item">
-                  <a
-                      class="nav-link nav-menu-button"
-                      href="<%= prefix_url("/apps") %>"
-                  >
-                      <%=
-                        branding('brandbar.apps_link.text') ||
-                          'Web Suite'
-                        %>
-                  </a>
+              <li class="nav-item dropdown">
+                  <div class="btn-group">
+                      <a
+                          class="nav-link nav-menu-button"
+                          href="<%= prefix_url("/apps") %>"
+                      >
+                          <%=
+                            branding('brandbar.apps_link.text') ||
+                              'Web Suite'
+                            %>
+
+                      </a>
+                      <a
+                          type="button"
+                          class="dropdown-toggle dropdown-toggle-split"
+                          data-toggle="dropdown"
+                          aria-haspopup="true"
+                          aria-expanded="false"
+                      >
+                          <span class="sr-only">Toggle Dropdown</span>
+                      </a>
+                      <div class="dropdown-menu dropdown-menu-right">
+                          <% app_items.each do |app| %>
+                              <a
+                                  href="<%= app[:path] %>"
+                                  class="dropdown-item nav-link nav-menu-button"
```

Comment:
Can you remove the `nav-link` and `nav-menu-button` classes.  The bootstrap documentation doesn't add `nav-link` to its nav dropdown items (https://getbootstrap.com/docs/4.6/components/navs/#pills-with-dropdowns).  I don't think we need to either.